examples/kubernetes/Dockerfile

```
@@ -0,0 +1,21 @@
+ARG BASE_IMAGE_NAME=vault.habana.ai/gaudi-docker/1.16.1/ubuntu22.04/habanalabs/pytorch-installer-2.2.2
+ARG BASE_IMAGE_TAG=latest
+
+FROM ${BASE_IMAGE_NAME}:${BASE_IMAGE_TAG} AS optimum-habana
```

```suggestion
FROM vault.habana.ai/gaudi-docker/1.16.1/ubuntu22.04/habanalabs/pytorch-installer-2.2.2:latest AS optimum-habana
```

Personal style, but I am wondering if you need to introduce these extra arguments?

---

My thought was that a user might have a different version of the Habana software on their machine, so instead of asking them to edit the Dockerfile, I defined it as an arg. Also, having a `BASE_IMAGE_NAME` and `BASE_IMAGE_TAG` allows for doing testing with a different base image (for example a pre-release image that is in a local/internal container registry). 

However, based on your other comment, it sounds like you would prefer something like this?
```
ARG GAUDI_SW_VER=1.16.1
ARG OS=ubuntu22.04
ARG TORCH_VER=2.2.2

FROM vault.habana.ai/gaudi-docker/${GAUDI_SW_VER}/${OS}/habanalabs/pytorch-installer-${TORCH_VER}:latest AS optimum-habana
```

---

@dmsuehir thanks, if you are using ARGS, then I usually find it easier to have them at the start, like here and use them everywhere.